site/multiDim/MultiDimDataPageSettingsPanel.tsx

```diff
@@ -123,9 +123,9 @@ const DimensionDropdown = (props: {
 export const MultiDimSettingsPanel = (props: {
     config: MultiDimDataPageConfig
     currentSettings: MultiDimDimensionChoices
-    updateSettings?: (currentSettings: MultiDimDimensionChoices) => void
+    onUpdateSettings: (currentSettings: MultiDimDimensionChoices) => void
```

Comment:
Renamed this to follow the React convention for naming event handler params. And we always pass the arg at the single call site, so it doesn't have to be optional.